include/rocksdb/db.h

```diff
@@ -353,21 +353,41 @@ class DB {
   // the i'th returned status will have Status::ok() true, and (*values)[i]
   // will store the value associated with keys[i].
   //
-  // (*values) will always be resized to be the same size as (keys).
+  // (*values) is required to be the same size as (keys).
```

Comment:
I do not think we can change the API so easily. Perhaps we can allow resize for std::string version of the API not to break the existing apps.